generators/languages/index.js

```diff
@@ -13,9 +13,39 @@ var LanguagesGenerator = generators.Base.extend({});
 
 util.inherits(LanguagesGenerator, scriptBase);
 
+var configOptions = {};
+
 module.exports = LanguagesGenerator.extend({
     constructor: function() {
         generators.Base.apply(this, arguments);
+
+        configOptions = this.options.configOptions || {};
+
+        // This makes it possible to pass `languages` by argument
+        this.argument('languagesArgument', {
```

Comment:
Sorry I meant that I tried call it languages though there was a problem setting it to props.languages later in the code line 81 for some reason.  I really tried it without any success. I think it may be since it is an argument but I'm not sure exactly.